Sources/Private/MainThread/LayerContainers/MainThreadAnimationLayer.swift

```diff
@@ -198,6 +198,13 @@ final class MainThreadAnimationLayer: CALayer, RootAnimationLayer {
     animationLayers.forEach { $0.displayWithFrame(frame: currentFrame, forceUpdates: true) }
   }
 
+  func getHierarchyKeypaths() -> [String] {
```

Comment:
An API like this would be more idiomatic:

```suggestion
  var allAnimationKeypaths: [String] {
```